src/js/components/form-elements/v2/Switch.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+
+import Translate from 'utils/Translate';
+
+import './style.scss';
+
+const Switch = ({ defaultValue, titles, onChange }) => {
+  const [value, changeValue] = useState(defaultValue);
+
+  const onChangeValue = () => {
+    changeValue((previousValue) => {
+      onChange?.(!previousValue);
+      return !previousValue;
+    });
+  };
+
+  return (
+    <div id="switch-container">
+      <label htmlFor="toggle" className="switch">
+        <input
+          id="toggle"
```

Comment:
I fell like this could use some sort of additional part of id passed in props :thinking: